src/providers/fault-count/cat-adi-part2/fault-count.cat-adi-part2.ts

```diff
@@ -35,11 +35,12 @@ export class FaultCountADIPart2Helper {
     const seriousFaultSumOfSimpleCompetencies = Object.keys(pickBy(seriousFaults)).length;
     const controlledStopSeriousFaults = (controlledStop && controlledStop.fault === CompetencyOutcome.S) ? 1 : 0;
     const eyesightTestSeriousFaults = (eyesightTest && eyesightTest.seriousFault) ? 1 : 0;
+    const seriousFaultFromVehicleChecks = vehicleChecks.seriousFault ? 1 : 0;
 
     const result =
       seriousFaultSumOfSimpleCompetencies +
       sumManoeuvreFaults(manoeuvres, CompetencyOutcome.S) +
-      FaultCountADIPart2Helper.getVehicleChecksByOutcomeFaultCountCatADIPart2(vehicleChecks, CompetencyOutcome.S) +
```

Comment:
does this method not work?

Reply:
It works fine, however the data for vehicle checks dont record serious faults within the outcome, it has its own boolean in the schema.

Reply:
ok, fair enough - makes sense!